shell.nix

```diff
@@ -0,0 +1,45 @@
+let 
+  pkgs = import <nixpkgs> {};
+  beku = pkgs.callPackage(pkgs.fetchFromGitHub {
```

Comment:
beku should also be pinnable with niv, so we handle it all in one place.

Reply:
Is that a change to make in the beku.py repo? Or here?

Reply:
Here. `niv add stackabletech/beku.py`, then grab it from `nix/sources.nix`.

Reply:
Ah ok cool, I think I understand.
Will ping you once I think I've done it.

Reply:
Thanks for your help @nightkr, resolved in 942b0f2dfb7548650ec5c5658d4357a3b25cff54